packages/frontend/src/app.css

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+:root {
+  color-scheme: dark;
+}
```

Comment:
shouldn't podman-desktop apply that automatically ?

I mean, it may not be the case but I guess it should be the role of Podman Desktop to do that

Reply:
Yes but it is not doing it.

We are absolutely not ready to support light theme so this is currently the best solution IMO to have a _decent_ ui.

I have noticed the following but I don't seems to see it working

https://github.com/containers/podman-desktop/blob/e4fcfe5ac143ef122b767581762867997193433f/packages/preload-webview/src/webview-preload.ts#L97

Reply:
we're not setting this value for the iframe

I'll add it so we don't need it there

Reply:
css changes removed

Reply:
created https://github.com/containers/podman-desktop/pull/6444 let me if it works for you